frontend/components/Dashboard/PostCard/MyFeedPostCard.tsx

```diff
@@ -1,32 +1,34 @@
 import React from 'react'
 import Link from 'next/link'
+// TODO: use Post type from generated graphql
+import { Post } from './postCardUtils'
 
-interface Props {
-  id: string
-  title: string
-  body: string
-  image?: string
-  author: string
+type Props = {
```

Comment:
Minor updates in this file, but this component, which is the post that appears in the my feed page, needs more work